app/views/user/settings.ctp

```diff
@@ -36,282 +36,318 @@ $this->set('title_for_layout', $pages->formatTitle(__('Settings', true)));
 </div>
 
 <div id="main_content">
-    <div class="module options">
+    <div class="options">
         <?php echo $form->create(null, array('action' => 'save_settings')); ?>
+        <div md-whiteframe="1" class="settings-form">
```

Comment:
This div is unnecessary. The attributes can be moved into the div above (`<div class="options">`).